app/src/main/java/com/hover/stax/faq/FaqFragment.kt

```diff
@@ -0,0 +1,93 @@
+package com.hover.stax.faq
+
+import android.os.Bundle
+import android.text.Html
+import android.text.method.LinkMovementMethod
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.hover.stax.R
+import com.hover.stax.databinding.FragmentFaqBinding
+import com.hover.stax.utils.UIHelper
+import com.hover.stax.utils.Utils
+import org.koin.androidx.viewmodel.ext.android.getViewModel
+
+class FaqFragment : Fragment(), FAQAdapter.Companion.SelectListener {
+
+    lateinit var binding: FragmentFaqBinding
```

Comment:
Fragments outlive their views so we need to clean up binding references when the view is destroyed. This should be initialized as a nullable variable.